releases/evm/2025-01-ITS-v2.1.0.md

```diff
@@ -118,8 +124,15 @@ node evm/interchainTokenFactory.js --action linkToken --destinationChain chain2
 node evm/its.js --action transferMintership --tokenAddress [tokenAddress] --minter [tokenManager]
 ```
 
+> [!NOTE]  
+> `tokenManager` address is different per token, which can be found on tx output from `registerCustomToken` action on chain
+
 - [ ]  Interchain Transfer (both ways)
 
 ```bash
 node evm/its.js --action interchainTransfer --destinationChain chain2 --tokenId [tokenId] --destinationAddress [recipient] --amount 1 --gasValue 0
 ```
+
+> [!NOTE]  
```

Comment:
Same, note shouldn't be at the end. Add a sentence as part of register custom token step to record the tokenId from the output